docs/orchestration/global-hooks/overview.mdx

```diff
@@ -0,0 +1,73 @@
+---
+title: Global Hooks
+sidebarTitle: "Overview"
+---
+<img
+  alt="Hook"
+  src="https://github.com/mage-ai/assets/blob/main/Hook.gif?raw=True"
+/>
+
+Global Hooks are a powerful feature in Mage that allows you to execute custom functions before or after an API request. These hooks can be used to perform additional operations, validate data, or integrate with external systems.
```

Comment:
Move this above the gif as well

Reply:
Moved this above as well, see picture from mintlify dev.
<img width="1759" alt="Screenshot 2024-06-19 at 9 14 04 AM" src="https://github.com/mage-ai/mage-ai/assets/92221031/b4788742-e164-4326-a0c5-11bf3cc0bd97">

Reply:
https://github.com/mage-ai/mage-ai/pull/5203#discussion_r1646574960

Reply:
Moved to description tested in mintlify dev
<img width="868" alt="Screenshot 2024-06-19 at 2 34 50 PM" src="https://github.com/mage-ai/mage-ai/assets/92221031/74eeba33-8cfa-4e1d-aa39-ba50c6f64af5">